### PR TITLE
Added solution for nuget

### PR DESCRIPTION
**Pull request by ajaygargdev**

### CODE REVIEW

File: Nuget/src/kmd.momentum.mea.client/Models/HealthReport.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Kmd.Momentum.Mea.Client.Models
```
Comment (ExploreMyCode) [inline Nuget/src/kmd.momentum.mea.client/Models/HealthReport.cs:7]: I Know this is auto-generated code. Also, there is not a big deal to have this code here.  Do we need this code?

File: Nuget/src/kmd.momentum.mea.client/Models/HealthReportEntry.cs [hunk 2]
```diff
@@ -0,0 +1,83 @@
+// <auto-generated>
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Kmd.Momentum.Mea.Client.Models
```
Comment (ExploreMyCode) [inline Nuget/src/kmd.momentum.mea.client/Models/HealthReportEntry.cs:7]: Same here. We may leave as it is. I just want to check what do you think?

File: Nuget/src/kmd.momentum.mea.client/Models/HealthReport.cs (hunk 1 above)
Comment (ajaygargdev) [reply Nuget/src/kmd.momentum.mea.client/Models/HealthReport.cs:7]: No, we don't need this code but the user can test the health of MEA

File: Nuget/src/kmd.momentum.mea.client/Models/HealthReportEntry.cs (hunk 2 above)
Comment (ajaygargdev) [reply Nuget/src/kmd.momentum.mea.client/Models/HealthReportEntry.cs:7]: No, we don't need this code but the user can test the health of MEA

**Pull request merged**